src/content/docs/new-relic-solutions/new-relic-one/ui-data/service-maps/service-maps.mdx

```diff
@@ -0,0 +1,165 @@
+---
+title: "Service map: Visualize your entities"
+tags:
+  - Understand dependencies
+  - Understand system dependencies
+  - Service maps
+metaDescription: 'New Relic''s service maps are visual, customizable representations of your application architecture.'
+redirects:
```

Comment:
These all look right to me! I didn't verify each individual redirect, but as long as you did the following (which it seems you did) it's good!

- Moved all redirects found in deleted docs to this doc
- Added deleted docs to this redirect list